### ISSUE TITLE
[Issue] `sm-local-configure` breaks on MacOS

### ISSUE THREAD

**Issue by yashvesikar** [aws-samples/sagemaker-ssh-helper#31]
I have tried following the directions to setup my [Local IDE integration with SageMaker Studio over SSH for PyCharm / VSCode](https://github.com/aws-samples/sagemaker-ssh-helper/tree/main#studio) and have run into an issue on my MacOS device.

When I run `sm-local-configure` I get the following output message:
```
> sm-local-configure

Darwin <MY COMPUTER> 22.5.0 Darwin Kernel Version 22.5.0: Mon Apr 24 20:53:19 PDT 2023; root:xnu-8796.121.2~5/RELEASE_ARM64_T6020 arm64
cat: /etc/issue: No such file or directory
cat: /etc/os-release: No such file or directory
Python 3.10.12
Password:
Sorry, try again.
Password:
sudo: apt-get: command not found
```

I believe the issue is this _install_unzip() function: 
https://github.com/aws-samples/sagemaker-ssh-helper/blob/049f97be4a41e165d71750e83ab1f6c9e43e5f96/sagemaker_ssh_helper/sm-helper-functions#L40-L46

which is called by `sm-local-configure`
https://github.com/aws-samples/sagemaker-ssh-helper/blob/049f97be4a41e165d71750e83ab1f6c9e43e5f96/sagemaker_ssh_helper/sm-local-configure#L16

I think this has partially been handled in the `_install_aws_cli` function with a seperate function for MacOS, If it would be helpful I can submit a PR to add a check to those methods to see if `unzip` and `curl` are already installed — which for MacOS I think they are by default.

The function works as expected (I think) if when i commented out those 2 lines and installed the package locally.

System: 
- Operating System: MacOS Ventura 13.4
- Processor: Apple M2 Pro

**Comment by ivan-khvostishkov** [comment]
Hi, @yashvesikar , thanks for spotting this bug! This is a regression in the script appeared in one of the recent releases. I will work on the new release asap.

**Comment by SanjanaKallol** [comment]
Any update?

**Comment by ivan-khvostishkov** [comment]
@SanjanaKallol , hi! I'm still working on the new release. In the meantime you can test changes from the commit 4f972f2a6 . Thank you for the patience.

**Comment by ivan-khvostishkov** [comment]
Fixed in [v1.12.0](https://github.com/aws-samples/sagemaker-ssh-helper/releases/tag/v1.12.0)
